### DISCUSSION TITLE
Carousel images for FLEX UI

### DISCUSSION THREAD

**Discussion by irazetti** [twilio-labs/paste#2886] [General] [answered]
We are building the new version of the plugins in FLEX UI and we are planning on using a "Carousel" for images in the cards and the details page. 
Do we have any carousel component in Paste? Or any specific recommendations?
Thanks

**Comment by gloriliale** [comment]
Hi @irazetti! We do not have a carousel component in Paste, but I'll get back to you with any recs the team might have. Thank you!

**Reply by gloriliale** [reply]
To make sure we have as much information as possible, could you send us any figma links or screenshots you're using?

**Comment by irazetti** [comment]
Sure!

So this big image on the top: ![Install](https://user-images.githubusercontent.com/104209586/207844872-3c9d1ad1-d368-4923-ba6d-6022e9d27cd9.jpg)

Then we were also thinking in the cards but I think the space is small and won't add that much value: ![FLEX UI 1 1](https://user-images.githubusercontent.com/104209586/207845190-707d9ac7-068a-458a-a5b8-15b2ca7929c1.jpg)

Thanks!

**Comment by gloriliale** [comment, marked as answer]
Thank you for these images @irazetti! I do agree with you that in the cards with such small real estate, and with so many cards, it wouldn't be that much of a value add.

For creating the carousel, the biggest thing to note would be ensuring there's a way to pause/stop it if it's animated. [More considerations to keep in mind here](https://inclusive-components.design/a-content-slider/)! 

Here's how other design systems have created them accessibly: https://www.lightningdesignsystem.com/components/carousel/

Let us know if that's helpful. We'll post any other thoughts our team might have in this thread if there are more.
